core/primitives/src/lib.rs

```diff
@@ -52,12 +52,23 @@ pub type BlockNumber = u32;
 /// The address format for describing accounts.
 pub type Address = sp_runtime::MultiAddress<AccountId, ()>;
 
+/// Hashing type
+pub type Hashing = sp_runtime::traits::BlakeTwo256;
+
 /// Block header type.
-pub type Header = sp_runtime::generic::Header<BlockNumber, sp_runtime::traits::BlakeTwo256>;
+pub type Header = sp_runtime::generic::Header<BlockNumber, Hashing>;
 
 /// Block type.
 pub type Block = sp_runtime::generic::Block<Header, sp_runtime::OpaqueExtrinsic>;
 
+/// This is the simplest bytecode to revert without returning any data.
+/// We will pre-deploy it under all of our precompiles to ensure they can be called from within
+/// contracts. (PUSH1 0x00 PUSH1 0x00 REVERT)
+pub const REVERT_BYTECODE: [u8; 5] = [0x60, 0x00, 0x60, 0x00, 0xFD];
```

Comment:
I think this should not be a part of core primitives.

Reply:
I hesitated for a moment too. Seem like a good place for `EVM_ADDR_PREFIX` rather than `REVERT_BYTECODE`. Or I'll throw it to the spec mod.

Reply:
Let's put it in the spec file.

Reply:
Okay. Let me fix it.